employees/common/strings.py

```diff
@@ -72,7 +74,9 @@ class AuthorReportListStrings(NotCallableMixin, Enum):
     HOURS_PER_MONTH_LABEL = _("Monthly hours")
     NO_REPORTS_MESSAGE = _("This employee has no reports to display.")
     RETURN_BUTTON_MESSAGE = _("Back to employee list")
-    PROJECTS_WORK_PERCENTAGE_HEADER = _("Projects work percentage")
+    PROJECTS_WORK_PERCENTAGE_HEADER = _("Monthly work stats per project")
+    HOURS_SUM_HEADER = _("Total hours")
+    HOURS_PERCENTAGE_HEADER = _("Percentage")
```

Comment:
I am aware this is slightly redundant, but right now I cannot find any way to concatenate enums, since they do not allow subclassing. Unless there is some convenient workaround I am not aware of, we'd probably have to change those to regular dicts + custom template tag for accessing dicts (if one wasn't implemented already).
But this is a major change that I think goes beyond this PR.

Reply:
I think that this is not so important. Maybe your idea with enums would be good for this purpose but for now it is not so important :)